src/resources/views/quiz.blade.php

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<html>
+    <head>
+    <title>{{$links->name}}</title> 
+    <link rel="stylesheet" href="{{ asset('reset.css') }}">
+    <link rel="stylesheet" href="{{ asset('style.css') }}">
+    <style></style>
+    </head>
+    <body>
+    <div class="question">
+    <!-- 設問のループ -->
+        @foreach ($questions as $question)
+            <h1 class="title">{{$loop->iteration}}．この地名は何て読む？</h1>
+            <div class="quiz-img-container">
+                <img src="../../img/{{$question->image}}" alt="">
```

Comment:
altは説明文を入れるようにするかalt属性毎削除してください
なるべく無指定ではなく画像の説明文的なのを入れてほしいです。
また src は 相対パス (../../) ではなくassetヘルパを利用した方がlaravelが効果的に取得してくれるのでそちら利用するように意識してほしいです。
```suggestion
                <img src="{{ asset(img/$question->image)}}" alt="">
```